pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -58,14 +59,28 @@ python3Packages.buildPythonApplication rec {
   # Executable in $out/bin is a symlink to $out/share/dupeguru/run.py
   # so wrapPythonPrograms hook does not handle it automatically.
   postFixup = ''
-    wrapPythonProgramsIn "$out/share/dupeguru" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/share/dupeguru" "$out $propagatedBuildInputs"
+  '';
+
+  doCheck = true;
+
+  checkInputs = [
+    pytestCheckHook
+    distro
+    pytest
+    send2trash
+    tox
```

Comment:
Do we really need tox? Did you try without?

Reply:
Fixed, we actually didn't.